src/components/fieldinfo/fieldinfo.js

```diff
@@ -58,7 +58,7 @@ angular.module('vlui')
 
           funcsPopup = new Drop({
             content: popupContent,
-            target: element.find('.type-caret')[0],
+            target: element.find('.fa-caret-down')[0],
```

Comment:
Not sure why you have to change this line.  It seems orthogonal to the description of the PR.